v2python/generate_shim.py

```diff
@@ -258,9 +259,10 @@ def __init__(self, args, out):
         self._objpaths = []
 
     def write_body(self):
+        template_rpath = 'template' + os.path.sep
```

Comment:
For platform neutral code, use pathlib

Reply:
Actually, I reverted the v2src/v2python changes as they're unused anyway.